R/autoplot.pop_data.R

```diff
@@ -51,11 +67,11 @@ age_scatter <- function(
   if (is.null(strata)) {
     plot1 <-
       object %>%
-      ggplot2::ggplot(ggplot2::aes(x = .data[[object %>% get_age_var()]],
-                                   y = .data[[object %>% get_value_var()]],
-                                   col = get(strata)
-                                   )
-                      )
+      ggplot2::ggplot(ggplot2::aes(
+        x = .data[[object %>% get_age_var()]],
+        y = .data[[object %>% get_value_var()]],
+        col = get(strata)
+      ))
```

Comment:
reformatting for ease of reading the code.

Reply:
good work here; see further simplifications in the patch PR I'm preparing